### ISSUE TITLE
README.md should include the start up process for tmux/gdb

### ISSUE THREAD

**Issue by mrmowgli** [kthready/gdb_tmux#2]
running tmux, or gdb or starting gdb insude of tmux doesn't appear to start up the window sessions. Perhaps an example of how it actually gets started with this setup?

**Comment by mrmowgli** [comment]
Looks like it can be started like so:
Modify the tmux.gdb.conf file and add:
```
new -s debugger # create new session named debugger
neww -n shell # create new window
```

to the beginning of the file. To start:

`tmux -f ~/.tmux.gdb.conf attach`

**Comment by Chamber6821** [comment]
Doesn't the `debug` script start tmux as it should?

**Comment by Chamber6821** [comment]
Yeah. `debug` just setup tmux window. But gdb you should run manually
